### PR TITLE
Add Indexing Group Sync APIs

### PR DESCRIPTION
**Pull request by snlraju-msft**
Ignite documentation for Graph Connectors externalGroup sync APIs.

### CODE REVIEW

File: api-reference/beta/api/externalconnection-post-groups.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: "Create externalGroup"
+description: "Create a new externalGroup object."
+author: "snlraju-msft"
+localization_priority: Normal
+ms.prod: "search"
+doc_type: apiPageType
+---
+
+# Create groups
```
Comment (Lauragra) [inline api-reference/beta/api/externalconnection-post-groups.md:10]: The H1 title here should be "Create externalGroup".

File: api-reference/beta/api/externalconnection-post-groups.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+Namespace: microsoft.graph
+
+Create a new externalGroup object.
```
Comment (Lauragra) [inline api-reference/beta/api/externalconnection-post-groups.md:14]: Link to the externalGroup resource.

**Review by jasonjoh — APPROVED** [review]

File: api-reference/beta/api/externalconnection-post-groups.md (hunk 1 above)
Comment (jasonjoh) [reply api-reference/beta/api/externalconnection-post-groups.md]: fixed

File: api-reference/beta/api/externalconnection-post-groups.md (hunk 2 above)
Comment (jasonjoh) [reply api-reference/beta/api/externalconnection-post-groups.md]: fixed

**Pull request merged**